rfc/rfc-38/rfc-38.md

```diff
@@ -89,7 +89,7 @@ createRelation(sqlContext, parameters, null)
 }
 ```
 
-For streaming writing and reading, DefaultSource#createSink and DefaultSource#createSink are called respectively.
+For streaming writing and reading, DefaultSource#createSource and DefaultSource#createSink are called respectively.
```

Comment:
might be `DefaultSource#createSink and DefaultSource#createSource` corresponds to `writing and reading`
or 
`For streaming reading and writing, DefaultSource#createSource and DefaultSource#createSink are called respectively.`